website_sale_product_subscribe_notify/models/res_config_settings.py

```
@@ -19,7 +20,12 @@ def get_values(self):
         res = super(ResConfigSettings, self).get_values()
         email_recipient_limit = self.env['ir.default'].get('res.config.settings', 'email_recipient_limit')
         res.update(
-           email_recipient_limit = email_recipient_limit,
+            email_recipient_limit = email_recipient_limit,
         )
         return res
 
+    @api.constrains('email_recipient_limit')
+    def _check_email_recipient_limit(self):
+        if self.email_recipient_limit <= 0:
+            raise Warning("Email Recipient should be greater than 0!")
```

Limit Email Recipient needs to be greater than 0.

---

raise ValidationError instead of Warning